2022-03-23 12:55:34,930 INFO    MainThread:533242 [wandb_setup.py:_flush():71] setting env: {}
2022-03-23 12:55:34,931 INFO    MainThread:533242 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-23 12:55:34,931 INFO    MainThread:533242 [wandb_init.py:_log_setup():371] Logging user logs to /home/vime/Desktop/entity_typing/entity_typing_experiments/wandb/run-20220323_125534-f9ammmhp/logs/debug.log
2022-03-23 12:55:34,931 INFO    MainThread:533242 [wandb_init.py:_log_setup():372] Logging internal logs to /home/vime/Desktop/entity_typing/entity_typing_experiments/wandb/run-20220323_125534-f9ammmhp/logs/debug-internal.log
2022-03-23 12:55:34,931 INFO    MainThread:533242 [wandb_init.py:init():404] calling init triggers
2022-03-23 12:55:34,932 INFO    MainThread:533242 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-03-23 12:55:34,932 INFO    MainThread:533242 [wandb_init.py:init():460] starting backend
2022-03-23 12:55:34,932 INFO    MainThread:533242 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-23 12:55:34,953 INFO    MainThread:533242 [backend.py:ensure_launched():216] starting backend process...
2022-03-23 12:55:34,965 INFO    MainThread:533242 [backend.py:ensure_launched():221] started backend process with pid: 533308
2022-03-23 12:55:34,968 INFO    MainThread:533242 [wandb_init.py:init():469] backend started and connected
2022-03-23 12:55:34,976 INFO    MainThread:533242 [wandb_init.py:init():533] updated telemetry
2022-03-23 12:55:35,009 INFO    MainThread:533242 [wandb_init.py:init():563] communicating current version
2022-03-23 12:55:35,574 INFO    MainThread:533242 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-23 12:55:35,574 INFO    MainThread:533242 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-23 12:55:35,750 INFO    MainThread:533242 [wandb_init.py:init():606] starting run threads in backend
2022-03-23 12:55:35,851 INFO    MainThread:533242 [wandb_run.py:_console_start():1810] atexit reg
2022-03-23 12:55:35,852 INFO    MainThread:533242 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-23 12:55:35,852 INFO    MainThread:533242 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-23 12:55:35,853 INFO    MainThread:533242 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-23 12:55:35,854 INFO    MainThread:533242 [wandb_init.py:init():633] run started, returning control to user process
2022-03-23 12:55:35,854 INFO    MainThread:533242 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-23 12:55:35,856 INFO    MainThread:533242 [wandb_run.py:_restore():1752] restore
2022-03-23 12:55:38,434 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 917
  total_bytes: 917
}

2022-03-23 12:55:38,737 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 917
  total_bytes: 917
}

2022-03-23 12:55:38,841 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 917
  total_bytes: 7288
}

2022-03-23 12:55:38,945 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 917
  total_bytes: 7288
}

2022-03-23 12:55:39,046 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3793
  total_bytes: 7288
}

2022-03-23 12:55:39,149 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7288
  total_bytes: 7288
}

2022-03-23 12:55:39,253 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7288
  total_bytes: 7288
}

2022-03-23 12:55:39,356 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7288
  total_bytes: 7288
}

2022-03-23 12:55:39,459 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7288
  total_bytes: 7288
}

2022-03-23 12:55:39,563 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7288
  total_bytes: 7288
}

2022-03-23 12:55:40,211 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7288
  total_bytes: 7288
}

2022-03-23 12:55:40,623 INFO    MainThread:533242 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7288
  total_bytes: 7288
}
local_info {
}

2022-03-23 12:55:41,815 INFO    MainThread:533242 [wandb_run.py:_append_files():2180] logging synced files
